interface/src/components/sheet-parser.js

```diff
@@ -1,22 +1,21 @@
-const parser = require('google-spreadsheets-key-parser');
 const Papa=require('papaparse')
 
-let arrayWithData=[];
 async function parseToCsv(googleSheetLink){
     return await new Promise((resolve,reject)=>{
-        const Tabletop= require('tabletop')
-        let sheetID = parser(googleSheetLink)["key"];
-         Tabletop.init({
-            key: sheetID,
-            callback:(data) => { resolve(showInfo(data)) },
-            simpleSheet: true
-        })    
+        googleSheetLink= 'https://cors-anywhere.herokuapp.com/' + googleSheetLink;
```

Comment:
Would it make sense to have these sheets locally available instead of Google? Perhaps fetching them as part of the docker build process?

Reply:
This might make it difficult for users to get data through sheets as they have to rebuild the whole project each time if they have new or different sheets.